[2025-01-22T11:38:15.819+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 24_runtime_environment_pools_dag.wait1 manual__2025-01-22T11:38:12.757851+00:00 [queued]>
[2025-01-22T11:38:15.835+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 24_runtime_environment_pools_dag.wait1 manual__2025-01-22T11:38:12.757851+00:00 [queued]>
[2025-01-22T11:38:15.836+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-22T11:38:15.864+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): wait1> on 2025-01-22 11:38:12.757851+00:00
[2025-01-22T11:38:15.873+0000] {standard_task_runner.py:60} INFO - Started process 3783 to run task
[2025-01-22T11:38:15.885+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '24_runtime_environment_pools_dag', 'wait1', 'manual__2025-01-22T11:38:12.757851+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/24_runtime_environment_pools.py', '--cfg-path', '/tmp/tmpe35c152d']
[2025-01-22T11:38:15.888+0000] {standard_task_runner.py:88} INFO - Job 70: Subtask wait1
[2025-01-22T11:38:15.926+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-22T11:38:16.008+0000] {task_command.py:423} INFO - Running <TaskInstance: 24_runtime_environment_pools_dag.wait1 manual__2025-01-22T11:38:12.757851+00:00 [running]> on host d3a2c9f62df6
[2025-01-22T11:38:16.244+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='24_runtime_environment_pools_dag' AIRFLOW_CTX_TASK_ID='wait1' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T11:38:12.757851+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-22T11:38:12.757851+00:00'
[2025-01-22T11:39:16.307+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-22T11:39:16.347+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=24_runtime_environment_pools_dag, task_id=wait1, execution_date=20250122T113812, start_date=20250122T113815, end_date=20250122T113916
[2025-01-22T11:39:16.428+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-22T11:39:16.455+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
